### PR TITLE
Sync latest changes on site

### PR DESCRIPTION
**Pull request by NaughtyChas**

### CODE REVIEW

File: docs/staff/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+|<img src="https://a.ppy.sh/7210268?1715498207.jpeg" width="100"/>|[Tomoe](https://osu.ppy.sh/users/7210268)|选图，测试|
+|<img src="https://a.ppy.sh/14721101?1717747082.jpeg" width="100"/>|[cdwcgt](https://osu.ppy.sh/users/14721101)|程序|
+|<img src="https://a.ppy.sh/30329943?1723016654.jpeg" width="100"/>|[Nosenso](https://osu.ppy.sh/users/30329943)|解说|
+
+
+
+
+
+<img src="https://a.ppy.sh/33887163?1720800005.jpeg" width="70"/> 
+
```
Comment (CloneWith) [inline docs/staff/index.md:46]: These lines seem duplicate.

Should we remove them?

**Comment by NaughtyChas** [comment]
Test build passed, Require review.
Pinging @CloneWith

File: docs/staff/index.md (hunk 1 above)
Comment (CloneWith) [reply docs/staff/index.md:46]: OK, I'll do that myself later...

**Review by CloneWith — APPROVED** [review]

File: docs/staff/index.md (hunk 1 above)
Comment (NaughtyChas) [reply docs/staff/index.md:46]: I suggest to leave these until we have fully transfer these information onto the new staffBoard design.

**Pull request merged**